common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/OAuth2Strategy.java

```diff
@@ -52,22 +52,28 @@
                 GenericTokenResponse extends TokenResponse,
                 GenericTokenResult extends TokenResult> {
 
-    protected String mTokenEndpoint;
-    protected String mAuthorizationEndpoint;
-    protected Uri mIssuer;
+    private String mTokenEndpoint;
```

Comment:
As these values are used, set, and accessed by subclasses, I'd prefer to keep them `protected`